src/main/java/lotto/model/LottoTickets.java

```diff
@@ -0,0 +1,31 @@
+package lotto.model;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoTickets {
+
+    private final List<LottoTicket> lottoTickets;
+
+    public LottoTickets(List<LottoTicket> lottoTickets) {
+        this.lottoTickets = lottoTickets;
+    }
+
+    public int getSize() {
+        return lottoTickets.size();
+    }
+
+    public LottoResult getWinningResult(LottoWinningNumbers winningNumbers) {
+        List<LottoRank> lottoRanks = lottoTickets.stream()
+                .map(lottoTicket -> lottoTicket.match(winningNumbers))
+                .collect(Collectors.toList());
+
+        return new LottoResult(lottoRanks, new PurchaseAmount(this.getSize() * LottoTicket.PRICE));
```

Comment:
내부적으로 직접 접근할 수 있는 상태이니 굳이 getSize 메서드를 쓸 필요가 있을까 싶어요. 🤔

```suggestion
        return new LottoResult(lottoRanks, new PurchaseAmount(this.lottoTickets.size() * LottoTicket.PRICE));
```

Reply:
반영하였습니다!